elastic/kibana
# Onboard "Custom Threshold" rule type with FAAD

Towards: https://github.com/elastic/kibana/issues/169867

This PR onboards "Custom Threshold" rule type with FAAD.

## To verify
Create a Custom Threshold rule by using a test index and DW. Set the `Role visibility` `metrics`.
When the rule runs, it generates an alert and saves it under `.internal.alerts-observability.threshold.alerts-default`.
The alert should be visible on `Observability > alerts` page as well.
